spec/models/level_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Level, :type => :model do
+	it { is_expected.to belong_to(:course) }
+
+	let(:level) { Level.new }
+	it "rejects negative exp_threshold" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Tab detected.